tidy3d/components/simulation.py

```diff
@@ -4040,14 +4055,247 @@ def subsection(
 
         return new_sim
 
-    def suggest_mesh_overrides(self, **kwargs) -> List[MeshOverrideStructure]:
-        """Generate a :class:.`MeshOverrideStructure` `List` which is automatically generated
-        from structures in the simulation.
+
+
+    def suggest_mesh_overrides(self,fraction_to_largest_cell,num_cells)-> Dict[Simulation]:
```

Comment:
Please explain what these parameters do in the docstring. Also should probably combine your later docstring with the original one.